src/enclave/rpc_sessions.h

```diff
@@ -192,7 +192,7 @@ namespace enclave
       // Caller authentication is done by each frontend by looking up
       // the caller's certificate in the relevant store table. The caller
       // certificate does not have to be signed by a known CA (nullptr).
-      cert = std::make_shared<tls::Cert>(nullptr, cert_, pk, tls::auth_default);
+      cert = std::make_shared<tls::Cert>(nullptr, cert_, pk, false);
```

Comment:
Nit: I'd make `false` the default and not need the extra arg here.

Reply:
Or perhaps better, make `true` the default and only override when you don't really care...

Reply:
Yeah, that was my plan too, but since the previous default wasn't `auth_required`, I'm getting a few verification errors that I need to sort out first.

Reply:
Right, this may be related to the errors I was seeing back then. would be good to understand what's the underlying cause.

Reply:
Done, `auth_required` is now the default.